Competition2018/src/main/java/competition/subsystems/elevator/commands/MoveToMaxHeightCommand.java

```diff
@@ -13,26 +13,30 @@
 
     ElevatorSubsystem elevator;
     PIDManager pid;
-
-    double max = 90;
+    double max;
 
     @Inject
     public MoveToMaxHeightCommand(ElevatorSubsystem elevator, PIDFactory pf) {
         this.elevator = elevator;
         pid = pf.createPIDManager("Elevator", 0.1, 0, 0);
         pid.setErrorThreshold(0.1);
     }
-
+    
     @Override
     public void initialize() {
         log.info("Initializing");
+        if (!elevator.isCalibrated()) {
+            log.warn("THE ELEVATOR WILL NOT BE ABLE TO RUN UNDER AUTOMATIC CONTROL!");
```

Comment:
See my comment about this in the other class

Reply:
done